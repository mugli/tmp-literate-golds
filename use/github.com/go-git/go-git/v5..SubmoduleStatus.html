<!DOCTYPE html>
<html>
<head>
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1">
<title>References: github.com/go-git/go-git/v5.SubmoduleStatus</title>
<link href="../../../../css/light-v0.3.2-preview.css" rel="stylesheet">
<script src="../../../../jvs/golds-v0.3.2-preview.js"></script>
<body onload="onPageLoad()"><div>

<pre><code><span style="font-size:x-large;">type <b><a href="../../../../pkg/github.com/go-git/go-git/v5.html">github.com/go-git/go-git/v5</a>.<a href="../../../../src/github.com/go-git/go-git/v5/submodule.go.html#line-317">SubmoduleStatus</a></b></span>

<span class="title">7 uses</span>

	github.com/go-git/go-git/v5 (current package)
		<a href="../../../../src/github.com/go-git/go-git/v5/submodule.go.html#line-55">submodule.go#L55</a>: func (s *Submodule) Status() (*<b>SubmoduleStatus</b>, error) {
		<a href="../../../../src/github.com/go-git/go-git/v5/submodule.go.html#line-64" class="path-duplicate">submodule.go</a><a href="../../../../src/github.com/go-git/go-git/v5/submodule.go.html#line-64">#L64</a>: func (s *Submodule) status(idx *index.Index) (*<b>SubmoduleStatus</b>, error) {
		<a href="../../../../src/github.com/go-git/go-git/v5/submodule.go.html#line-65" class="path-duplicate">submodule.go</a><a href="../../../../src/github.com/go-git/go-git/v5/submodule.go.html#line-65">#L65</a>: 	status := &amp;<b>SubmoduleStatus</b>{
		<a href="../../../../src/github.com/go-git/go-git/v5/submodule.go.html#line-304" class="path-duplicate">submodule.go</a><a href="../../../../src/github.com/go-git/go-git/v5/submodule.go.html#line-304">#L304</a>: type SubmodulesStatus []*<b>SubmoduleStatus</b>
		<a href="../../../../src/github.com/go-git/go-git/v5/submodule.go.html#line-317" class="path-duplicate">submodule.go</a><a href="../../../../src/github.com/go-git/go-git/v5/submodule.go.html#line-317">#L317</a>: type <b>SubmoduleStatus</b> struct {
		<a href="../../../../src/github.com/go-git/go-git/v5/submodule.go.html#line-325" class="path-duplicate">submodule.go</a><a href="../../../../src/github.com/go-git/go-git/v5/submodule.go.html#line-325">#L325</a>: func (s *<b>SubmoduleStatus</b>) IsClean() bool {
		<a href="../../../../src/github.com/go-git/go-git/v5/submodule.go.html#line-336" class="path-duplicate">submodule.go</a><a href="../../../../src/github.com/go-git/go-git/v5/submodule.go.html#line-336">#L336</a>: func (s *<b>SubmoduleStatus</b>) String() string {
</code></pre><pre id="footer">
<table><tr><td><img src="../../../../png/go101-twitter.png"></td>
<td>The pages are generated with <a href="https://go101.org/article/tool-golds.html"><b>Golds</b></a> <i>v0.3.2-preview</i>. (GOOS=darwin GOARCH=amd64)
<b>Golds</b> is a <a href="https://go101.org">Go 101</a> project developed by <a href="https://tapirgames.com">Tapir Liu</a>.
PR and bug reports are welcome and can be submitted to <a href="https://github.com/go101/golds">the issue list</a>.
Please follow <a href="https://twitter.com/go100and1">@Go100and1</a> (reachable from the left QR code) to get the latest news of <b>Golds</b>.</td></tr></table></pre>