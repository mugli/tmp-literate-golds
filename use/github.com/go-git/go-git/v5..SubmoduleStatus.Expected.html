<!DOCTYPE html>
<html>
<head>
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1">
<title>References: github.com/go-git/go-git/v5.SubmoduleStatus.Expected</title>
<link href="../../../../css/light-v0.3.2-preview.css" rel="stylesheet">
<script src="../../../../jvs/golds-v0.3.2-preview.js"></script>
<body onload="onPageLoad()"><div>

<pre><code><span style="font-size:x-large;"><b><a href="../../../../pkg/github.com/go-git/go-git/v5.html">github.com/go-git/go-git/v5</a>.<a href="../../../../src/github.com/go-git/go-git/v5/submodule.go.html#line-317">SubmoduleStatus</a>.<a href="../../../../src/github.com/go-git/go-git/v5/submodule.go.html#line-320">Expected</a></b></span><span style="font-size: large;"><i> (field)</i></span>

<span class="title">5 uses</span>

	github.com/go-git/go-git/v5 (current package)
		<a href="../../../../src/github.com/go-git/go-git/v5/submodule.go.html#line-75">submodule.go#L75</a>: 		status.<b>Expected</b> = e.Hash
		<a href="../../../../src/github.com/go-git/go-git/v5/submodule.go.html#line-320" class="path-duplicate">submodule.go</a><a href="../../../../src/github.com/go-git/go-git/v5/submodule.go.html#line-320">#L320</a>: 	<b>Expected</b> plumbing.Hash
		<a href="../../../../src/github.com/go-git/go-git/v5/submodule.go.html#line-326" class="path-duplicate">submodule.go</a><a href="../../../../src/github.com/go-git/go-git/v5/submodule.go.html#line-326">#L326</a>: 	return s.Current == s.<b>Expected</b>
		<a href="../../../../src/github.com/go-git/go-git/v5/submodule.go.html#line-356" class="path-duplicate">submodule.go</a><a href="../../../../src/github.com/go-git/go-git/v5/submodule.go.html#line-356">#L356</a>: 	return fmt.Sprintf("%c%s %s%s", status, s.<b>Expected</b>, s.Path, extra)
		<a href="../../../../src/github.com/go-git/go-git/v5/worktree_status.go.html#line-194">worktree_status.go#L194</a>: 			o[s.Path] = s.<b>Expected</b>
</code></pre><pre id="footer">
<table><tr><td><img src="../../../../png/go101-twitter.png"></td>
<td>The pages are generated with <a href="https://go101.org/article/tool-golds.html"><b>Golds</b></a> <i>v0.3.2-preview</i>. (GOOS=darwin GOARCH=amd64)
<b>Golds</b> is a <a href="https://go101.org">Go 101</a> project developed by <a href="https://tapirgames.com">Tapir Liu</a>.
PR and bug reports are welcome and can be submitted to <a href="https://github.com/go101/golds">the issue list</a>.
Please follow <a href="https://twitter.com/go100and1">@Go100and1</a> (reachable from the left QR code) to get the latest news of <b>Golds</b>.</td></tr></table></pre>